tests/saw/sike_r2/proof/shake.saw

```diff
@@ -0,0 +1,86 @@
+// cshake.saw Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nit: filename in comment (`cshake.saw`) doesn't match actual filename (`shake.saw`)